repos/fdbt-site/src/pages/api/manageFareDayEnd.ts

```diff
@@ -0,0 +1,31 @@
+import { NextApiResponse } from 'next';
+import { redirectTo, redirectToError, getAndValidateNoc } from './apiUtils';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { NextApiRequestWithSession } from '../../interfaces';
+import { GS_FARE_DAY_END_ATTRIBUTE } from '../../constants/attributes';
+import { removeExcessWhiteSpace, isValid24hrTimeFormat } from './apiUtils/validator';
+import { upsertFareDayEnd } from '../../data/auroradb';
+import { fareDayEndInputId } from '../manageFareDayEnd';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    try {
+        const fareDayEnd: string | undefined = removeExcessWhiteSpace(req.body.fareDayEnd);
```

Comment:
This function always returns a string, do we need to have the " | undefined " part?

Reply:
👍  will remove

Reply:
Done